README.md

```diff
@@ -19,12 +19,12 @@ A curated list of awesome packages and frameworks for implementing javascript ap
 
 ### Web
 
-* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (Windows, macOS, Linux)
-* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (macOS only)
+* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (cross-platform)
```

Comment:
What's the difference between "cross-platform" and "Windows, macOS, Linux"?

Reply:
more people search "cross platform desktop framework" than "windows mac and linux desktop framework"

Reply:
I don't think so. Let's keep it as is. Imagine I want to search "macOS".

Reply:
I'll edit this a bit and make a new PR then?

Reply:
Sure 👍